eclair-core/src/main/resources/reference.conf

```diff
@@ -382,6 +383,7 @@ eclair {
   blockchain-watchdog {
     // all available blockchain watchdog sources
     // you can remove items from this list to disable failing or non-needed sources
+    tolerance = 7
```

Comment:
You should probably move that below the `sources` list, now we don't understand what the documentation above applies to. You should also add a comment to explain what that field does.

I would rename it to `missing-blocks-threshold` which is more explicit (and please rename the corresponding field in the code as well).

Reply:
Corrected